packages/components/src/internal/util/utils.ts

```diff
@@ -335,6 +336,10 @@ export function similaritySortFactory(token: string, caseSensitive?: boolean): (
     };
 }
 
+export const isLoading = (loadingState: LoadingState): boolean => {
+    return loadingState === LoadingState.INITIALIZED || loadingState === LoadingState.LOADING;
+};
```

Comment:
Should we make use of this within `QueryModel.ts` at this time? or do that in a future PR.

Reply:
see isLoading(), isLoadingCharts(), and isLoadingSelections()

Reply:
It looks like @labkey-alan and I did this simultaneously in separate branches. I'm going to leave my declaration as-is and it will be replaced by the "public" version declared by him.